pyanaconda/core/glib.py

```diff
@@ -35,7 +35,7 @@
                                IOCondition, IOChannel, SpawnFlags, \
                                MAXUINT
 
-__all__ = ["create_main_loop", "create_new_context",
+__all__ = ["create_main_loop", "create_new_context", "mainloop_run"
```

Comment:
There is a missing colon at the end of line

Reply:
oops, I will fix this

Reply:
Addressed in https://github.com/rhinstaller/anaconda/pull/4322